app/build.gradle

```diff
@@ -76,6 +76,10 @@ android {
     androidExtensions {
         experimental = true
     }
+
+    buildFeatures {
+        viewBinding = true
+    }
```

Comment:
i think we can remove this from this PR as it has already been merged in `migrate_to_view_binding` branch